code_benches/run.py

```diff
@@ -76,13 +96,13 @@ def __str__(self):
                               "-Wno-shift-negative-value"]),
     Program("custom_matrix_multiply", [], 2 ** 14),
     Program("custom_memcmp", [], 2 ** 14),
-    Program("custom_sqlite", [], 2 ** 15),
+    Program("custom_sqlite", [], 2 ** 15, custom_arguments=["-pthreads", "-ldl"]),
```

Comment:
Not sure if `-pthreads` should be used though, as we don't support multi-threading within a Wasm application.

Reply:
Ah, it looks like sqlite was just using -pthreads for the mutex definitions. Alternatively it looks like I can use -DSQLITE_MUTEX_NOOP, will change.